branding_sample_trial/services/application.yml

```diff
@@ -0,0 +1,5 @@
+management:
```

Comment:
Should the name (and location) of the logo file be configured in one of the YML files that control the deployment?

Reply:
it will be taken from the images directory in the trial directory but won't be a configurable value).
so each trial will have a logo. you simply put it in the directory